Week5/solution-week-5.py

```diff
@@ -0,0 +1,32 @@
+import csv
+
+with open('student.csv','w',newline='')as f:
+    w=csv.writer(f)
+    header=['sid','sname','city','contact']
+    w.writerow(header)
+
+#Insert 5 Records Directly
+    row=[[1,'om','kedarnath',9876567890],
+         [2,'sai','shirdi',1231231230],
+         [3,'ram','ayodhya',1234567865],
+         [4,'kishan','vrundavan',2345670763],
+         [5,'ravan','lanka',8978655544]]
+    w.writerows(row)

```

Comment:
What if I want to pass header with  this row variable if any other way possible please suggest me